### PR TITLE
Update README.md

### LINKED ISSUE #4: add new look to the readme
### Is your feature request related to a problem? Please describe
I have given your README a completely new look and added simplified text that is highly useful for new contributors. I incorporated the text you provided and made sure it enhances the overall appeal of the document. Please review my changes and let me know if there are any issues. If you have any concerns, feel free to assign me to the relevant issue, and I will address it as soon as possible.

### PR DESCRIPTION
**Pull request by Ayush-Tibrewal**
GIVEN NEW LOOK

## ADDED BAGES AND WRITE THE WHOLE THING NEWLY 

Fixes #4 

## Dependencies

_Mention any dependencies/packages used_

## Future Improvements

_Mention any improvements to be done in future related to any file/feature_

## Mentions

_Mention and tag the people
HTML AND CSS_

## Screenshots of relevant screens
image

## Developer's checklist

- [y] My PR follows the style guidelines of this project
- [y ] I have performed a self-check on my work

**If changes are made in the code:**

- [ y] I have followed the [coding guidelines](https://google.github.io/styleguide/jsguide.html)
- [ y] My changes in code generate no new warnings
- [ y] My changes are breaking another fix/feature of the project
- [ ] I have added test cases to show that my feature works
- [y ] I have added relevant screenshots in my PR

### CODE REVIEW

**Comment by Ayush-Tibrewal** [comment]
image
@advaita-saha please check

**Comment by Ayush-Tibrewal** [comment]
@advaita-saha please check

**Comment by advaita-saha** [comment]
Please go through the project and understand what are the features in EthUnwrapped and update it properly

File: README.md [hunk 1]
```diff
@@ -1,17 +1,63 @@
...
+## Introduction
+
+eth-unwrapped is a curated repository that provides information and resources about various unwrapped tokens in the Ethereum ecosystem. Unwrapped tokens are traditional assets represented on the Ethereum blockchain through smart contracts, enabling greater interoperability and decentralized finance (DeFi) applications. 💫
+
+## Features
+
+- 📚 Extensive Collection: Browse through a wide range of unwrapped tokens, including stablecoins, commodities, indices, and more.
+- ℹ️ Token Details: Access detailed information about each token, such as contract addresses, supply, decimals, and associated links.
+- 💻 Developer-Friendly: Find resources for integrating unwrapped tokens into your smart contracts and applications.
+- 🤝 Community Contributions: Contribute to the repository by suggesting new tokens or providing updates to existing ones.
```
Comment (advaita-saha) [inline README.md:32]: wrong, please go through the project

File: README.md [hunk 2]
```diff
@@ -1,17 +1,63 @@
-# EthUnwrapped
 
-Get you own ETH statistics unwrapped!
+# 🎁 Eth-unwrapped
 
-## About
-EthUnwrapped is a web application that analizes the Ethereum blockchain and provides a statistics about your usage of the blockchain.
+Welcome to Eth-unwrapped, a comprehensive collection of unwrapped Ethereum tokens! 🚀
```
Comment (advaita-saha) [inline README.md:4]: revert to the original statement, what you have written is wrong. This is not what ethunwrapped is doing

Comment (Ayush-Tibrewal) [reply README.md:4]: ok i am going through you project thoroughly please wait @advaita-saha

**Comment by Ayush-Tibrewal** [comment]
@advaita-saha done the change  
sorry for the mistakes 
ready to resolve further issues .

**Pull request merged**